src/Avalonia.Themes.Default/WindowNotificationManager.xaml

```diff
@@ -0,0 +1,45 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    <Style Selector="WindowNotificationManager">
+        <Setter Property="Margin" Value="0 0 8 8"/>
+        <Setter Property="Template">
+            <ControlTemplate>
+                <StackPanel Name="PART_Items">
+                    <StackPanel.DataTemplates>
+                        <DataTemplate DataType="INotification">
+                            <StackPanel Spacing="8" Margin="12">
+                                <TextBlock DockPanel.Dock="Top" Text="{Binding Title}" FontWeight="Medium" />
```

Comment:
I don't think `DockPanel.Dock` will do anything here, the parent is a `StackPanel`.